pkg/controller/unifiedpushserver/postgresql.go

```diff
@@ -41,9 +41,11 @@ func newPostgresqlSecret(cr *pushv1alpha1.UnifiedPushServer) (*corev1.Secret, er
 	return &corev1.Secret{
 		ObjectMeta: objectMeta(cr, "postgresql"),
 		StringData: map[string]string{
-			"database-name":     "unifiedpush",
-			"database-user":     "unifiedpush",
-			"database-password": databasePassword,
+			"POSTGRES_DATABASE":  "unifiedpush",
```

Comment:
The [integr8ly/backup-container-image](https://github.com/integr8ly/backup-container-image) needs this to be in a specific format. Also, if we do decide to use PostgreSQL 10 (there's a different comment about that in another file in this PR), then we need to specify `POSTGRES_VERSION` in here too.